tests/README.md

```diff
@@ -0,0 +1,29 @@
+# SDO ANALYSIS TESTS
+
+---
+
+## HOW TO RUN
+- Attach to the ```yc-sdo-analysis``` container
+- Set all the needed environment variables using: ```export YANGCATALOG_CONFIG_PATH=$PWD/tests/resources/test.conf && export PYTHONPATH=$PWD/bin:$PYTHONPATH && export VIRTUAL_ENV=$PWD```
+- Now you're able to run all the tests locally:
+  - To run all the tests: ```pytest```
+  - To run the tests in a particular file: ```pytest tests/test_file_name.py```
+  - To run the particular test class or the method of the test class: ```pytest tests/test_file_name.py::TestClass::test_method```
+
+---
+
+## COVERAGE
+For test coverage we are using [Coverage.py](https://coverage.readthedocs.io/en/6.5.0/). Here you can see which modules are currently covered in tests (**do not forget to update this information every time you create/update/delete a test; follow the steps listed below, run coverage for all tests, and paste the screenshot of the covered scripts from the ```tests/htmlcov/index.html```**):
+![img.png](tests_coverage.png)
+To see all the information about the tests coverage locally follow these steps:
```

Comment:
There needs to be an empty line here, otherwise it gets rendered on the same line as the image.